### DISCUSSION TITLE
Query Builder for influx_V2 , influxQL FLUX

### DISCUSSION THREAD

**Discussion by peterl3233** [grafana/grafana#38413] [Data Sources]
The Grafana menu query builder for influxdb 1.x query language "influxQL" is a simply a beautiful thing ... easy to setup, change and adjust.  

After upgrades, I see that it is no longer available for Influxdb_v2 using FLUX.  As we know, influxdb_v2 uses "buckets" (etc) and Influx_V1.x uses more traditional "Databases" (etc).

Feature Request: is to include a FLUX compatible Query Builder similar to the  as a basic feature of Grafana.

**Comment by Totalus** [comment]
InfluxDB 2 provides an integrated dashboard with a query builder. You can build your queries directly in the influxDB dashboard and then copy-paste the resulting flux script to your Grafana query. I find this workflow easy enough for myself. I don't know if having an integrated query builder in Grafana would make it significantly more convenient. Maybe users do not always have access to the influxDB dashboard though.

**Comment by pprkut** [comment]
I think this is a duplicate of #43848

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
